ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -160,6 +172,26 @@ message GetUserResponse {
     User user = 1;
 }
 
+// https://cloud.google.com/apis/design/standard_methods#update
+// https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#fieldmask
+message UpdateUserRequest {
+    // The user to update.
+    // Modifiable fields:
+    // 1. primary_party
+    // 2. is_deactivated
+    User user = 1;
+
+    // Specifies the fields of the user resource to be updated.
+    // At least one field must be specified.
+    // Required
+    FieldMask update_mask = 2;
```

Comment:
Why have `primary_party` and `is_deactivated` modified one way, but properties modified a different way?

Reply:
I am mulling over this as well. The current approach is the consequence of the usage of the property bag that in itself is an update bundle where individual elements need to be marked for update/delete.

More radically and simply, how about using the replace semantics, where the entire set of the properties is replaced with a new set. This would be represented with a single bit in the update mask. It would be up to the user not to lose the full content of the set. I think it is actually easier for them to implement than constructing separate add/delete list of properties.

Reply:
Changed to what @mziolekda suggested, i.e. making properties part of the user resource and updating them (with 'replace-all' semantics) with the UpdateUser call.

Made the same change on the parties

Reply:
Thanks for that. I like the simplicity of it, but let's hear from @da-tanabe what he thinks about the ergonomics of this from the UI perspective

Reply:
What about making the field mask semantics such that it allows setting specific properties by referencing them in the field_mask? So the set only the `description` property one would use the `properties.description` field mask. Whereas, if one wanted to set all properties one would use `properties` as the only mask. Using both would not be allowed. Setting a property that is not present in the submitted user would delete the property.

Reply:
This semantics would obviously have to be explained here.

Reply:
It looks good right now, like Simon has mentioned we need to document here how the field mask descends into the annotations in the object meta

Reply:
For the record: settled at 'replace-all' semantics for the `ObjectMeta.annotations` map.